action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'setup-testspace'
+description: 'Action to install and configuration the Testspace client'
+author: 'S2 Technologies, Inc.'
+inputs:
+  subdomain:  
```

Comment:
Why `subdomain`? The simpler (as word and purpose) `domain` is a better choice to me.

Reply:
Isn't `subdomain` the semantically correct term? We also use `subdomain` throughout our help.

Reply:
The option should be `domain`, but allow specifying just the "subdomain" when using `testspace.com` as a root domain. We should also consider enterprise, where the root domain could be different (e.g. `dolby.testspace.com`).

Reply:
Why not option `subdomain`, allowing specifying the "domain" optionally. The use case is for cloud users that are leveraging *GitHub Actions*, not enterprise. BTW I don't know when actions will be available for Enterprise. 

But if `domain` was used, any thoughts on how to explain only the org-name is required?  Using the `action.yml`:
<pre>
inputs:
  domain:  
    description: 'Testspace domain, only the subdomain is required`
    required: true
</pre>